src/Components/Members/Create/index.jsx

```diff
@@ -0,0 +1,234 @@
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          firstName,
+          lastName,
+          dni,
+          phone,
+          email,
+          city,
+          birthDay,
+          postalCode,
+          isActive,
+          membership
+        })
+      });
+
+      const data = await newMembers.json();
+      setMessage(data.message);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    addMember({
+      firstName,
+      lastName,
+      dni,
+      phone,
+      email,
+      city,
+      birthDay,
+      postalCode,
+      isActive,
+      membership
+    });
+    setFirstName('');
+    setLastName('');
+    setDni('');
+    setPhone('');
+    setEmail('');
+    setCity('');
+    setBirthday('');
+    setPostalCode('');
+    setIsActive(true);
+    setMembership('Classic');
+    setShowModal(true);
+  };
+
+  return (
+    <div className={styles.formContainer}>
+      <Modal
+        onClose={() => setShowModal(false)}
+        isOpen={showModal}
+        title={message}
+        success={true}
+      />
+      ;
+      <div>
+        <h3>Create new member</h3>
+        <form onSubmit={(e) => handleSubmit(e)}>
+          <div>
+            <Input
+              labelText={'Name'}
+              type={'text'}
+              placeholder={'Name'}
+              value={firstName}
+              onChangeInput={handleFirstName}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'Surname'}
+              type={'text'}
+              placeholder={'Surname'}
+              value={lastName}
+              onChangeInput={handleLastName}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'DNI'}
+              type={'text'}
+              placeholder={'DNI'}
+              value={dni}
+              onChangeInput={handleDniChange}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'Phone'}
+              type={'text'}
+              placeholder={'ex: 096513178'}
+              value={phone}
+              onChangeInput={handlePhoneChange}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'Email'}
+              type={'text'}
+              placeholder={'robertomariaoverdrive@soybostero.edu'}
+              value={email}
+              onChangeInput={handleEmailChange}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'City'}
+              type={'text'}
+              placeholder={'Your city'}
+              value={city}
+              onChangeInput={handleCityChange}
+            />
+          </div>
+          <div>
+            <Input
+              labelText={'Birthday'}
+              type={'text'}
+              placeholder={'01/01/2000'}
```

Comment:
This is not the correct format for the birthday. If you try to use it, an error will be sent.

Reply:
Although the date format is depicted this way here, it is sent o the server correctly